multimodal/src/autogluon/multimodal/optimization/lit_module.py

```diff
@@ -41,7 +41,7 @@ def __init__(
         custom_metric_func: Callable = None,
         test_metric: Optional[torchmetrics.Metric] = None,
         efficient_finetune: Optional[str] = None,
-        trainable_param_names: Optional[List[str]] = None,
+        trainable_param_names: Optional[List] = [],
```

Comment:
We should still use `None` as the default value. Also, should it be `List[str]`?

Reply:
We can cast None to `[]` later .